### ISSUE TITLE
Zeus interaction - Unit - Repair

### ISSUE THREAD

**Issue by prisonerMO** [acemod/ACE3#9675]
**Mods (complete and add to the following information):**
- **Arma 3:** `2.14 stable`
- **CBA:** `3.16.1 Stable` 
- **ACE3:**  `3.16.1 Stable` 

**Description:**
Zeus Action - Repair

on vehicle , it will transform vehicle position where you click repair , if you have enabled active interaction by click.
It's blocked if on current position is another vehicle, but on free space it will change position of selected vehicle.

**Steps to reproduce:**
1.Launch Arma 3
2. Open editor , random map.
3.Select from Ace Action CBA settings -> enable interaction use on by click
4.Add game master module and #adminLogged
5.Add unit and empty vehicles on map.
6.Launch mission
7.Enter to Curator (Zeus)
8.Select Vehicle
9.Select from Zeus interactions -> Unit -> Repair
10.It will transform Vehicle to position of Click (when used Repair)

**Expected behavior:**
It will transform curator selected Vehicle to position of Click (when used Repair)

**Where did the issue occur?**
- Dedicated / Self-Hosted Multiplayer / Singleplayer / Editor (Singleplayer) / Editor (Multiplayer)

**Log Files:**
- No RPT errors

**Additional context:**
https://youtu.be/ZwMTNxHtFf4?si=ZImRQNBYPTVdx99T

**Comment by Dystopian** [comment]
IIRC it's vanilla zeus bug. Better use https://github.com/zen-mod/ZEN.

**Comment by johnb432** [comment]
I turned off `ace_interact_menu_actionOnKeyRelease` and I wasn't able to reproduce the issue with CBA + ACE only. Is this still an issue?

**Comment by johnb432** [comment]
> [...]

Video is private.

---

I've tried to reproduce on my desktop: Changed from profiling to stable, used both dev and stable releases of both CBA and ACE, but I can't reproduce it at all.
I tested it on my laptop and there it was possible, but only every 1 in 15 or so, which isn't sufficient for trying to find a fix.

Given the framedrops in the first video, my guess is that you are running in lower end/older hardware? My theory is that this bug can only happen on lower performing system - at least it would line with the case of my laptop, which has an i7-4700HQ and a GTX 860M.

Initially I thought it could be the mouse double clicking, given my laptop's mouse does that, but I tried with a better mouse and the problem persisted.
